### PR TITLE
Added organizations create subcommand

### PR DESCRIPTION
**Pull request by alijared**
This command is only useable by super users

### CODE REVIEW

File: croud/__main__.py [hunk 1]
```diff
@@ -39,7 +47,7 @@ def main():
     commands: dict = {
         "me": {
             "description": "Prints the current logged in user",
-            "extra_args": [output_fmt_arg, region_arg],
+            "extra_args": [output_fmt_arg],
```
Comment (mikethebeer) [inline croud/__main__.py]: I think this change has nothing to do with this feature. Please make a separate PR if needed.

File: tests/unit_tests/test_util.py [hunk 2]
```diff
@@ -18,17 +18,21 @@
 # software solely pursuant to the terms of the relevant commercial agreement.
 
 import unittest
+from argparse import Namespace
```
Comment (mikethebeer) [inline tests/unit_tests/test_util.py]: where is it used?

Comment (alijared) [reply tests/unit_tests/test_util.py]: https://github.com/crate/croud/pull/40/files#diff-7d5dce122957336cd058de719c4ed7f9R96

https://github.com/crate/croud/pull/40/files#diff-7d5dce122957336cd058de719c4ed7f9R102
